src/dg/strong_dg.cpp

```diff
@@ -1592,7 +1592,7 @@ void DGStrong<dim,nstate,real,MeshType>::assemble_face_term_strong(
             soln_state_ext[istate] = soln_at_surf_q_ext[istate][iquad];
             for(int idim=0; idim<dim; idim++){
                 aux_soln_state_int[istate][idim] = aux_soln_at_surf_q_int[istate][idim][iquad];
-                aux_soln_state_ext[istate][idim] = aux_soln_at_surf_q_int[istate][idim][iquad];
+                aux_soln_state_ext[istate][idim] = aux_soln_at_surf_q_ext[istate][idim][iquad];
```

Comment:
Any test that would catch this?

Reply:
Well ahead of you :) I'm adding an energy test for diffusion for the energy conservation case, that should catch it. I tried using the 2D NS OOA test, but it preserved the orders with and without the bug.

Reply:
Resolving this conversation; will be addressed in issue #187